socialdistribution/profiles/migrations/0010_auto_20200310_0414.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.1.5 on 2020-03-10 04:14
```

Comment:
I just found we are keeping push those file in `migrations`. I don't know is there a way we can avoid them. or automatically delete the old one?